src/main/java/bhs/devilbotz/utils/ShuffleboardManager.java

```diff
@@ -68,13 +68,17 @@ private void initAutoModeChooser() {
     autoModeChooser.addOption("Routine 3: Score and Mobility", AutonomousModes.SCORE_AND_MOBILITY);
     autoModeChooser.addOption("Routine 4: Dock and Engage", AutonomousModes.DOCK_AND_ENGAGE);
     autoModeChooser.addOption(
-        "Routine 5: Mobility, Dock, and Engage", AutonomousModes.MOBILITY_DOCK_AND_ENGAGE);
+        "Routine 5: Mobility, Dock, and Engage Wall Side",
+        AutonomousModes.MOBILITY_DOCK_AND_ENGAGE_WALL_SIDE);
     autoModeChooser.addOption(
-        "Routine 6: Score, Dock, and Engage", AutonomousModes.SCORE_DOCK_AND_ENGAGE);
+        "Routine 6: Mobility, Dock, and Engage Human Side",
+        AutonomousModes.MOBILITY_DOCK_AND_ENGAGE_HUMAN_SIDE);
```

Comment:
Talk to Adi about how he wants the routines named/organized to avoid confusion. Also, for the standalone mobility routine you should measure the ideal distance and use that as the default.

Reply:
These names were suggested by the strategy team. We can always change it after if Adi wants to.